mysql/mysqldump.go

```diff
@@ -274,18 +302,19 @@ func processAlterTable(conv *internal.Conv, stmt *ast.AlterTableStmt) {
 				processConstraint(conv, tableName, item.Constraint, "ALTER TABLE")
 				conv.SchemaStatement(NodeType(stmt))
 			case ast.AlterTableModifyColumn:
-				colname, col, isPk, err := processColumn(conv, tableName, item.NewColumns[0])
+				colname, col, constraint, err := processColumn(conv, tableName, item.NewColumns[0])
 				if err != nil {
 					logStmtError(conv, stmt, err)
 					return
 				}
 				conv.SrcSchema[tableName].ColDefs[colname] = col
-				if isPk {
+				if constraint.isPk {
 					ctable := conv.SrcSchema[tableName]
 					checkEmpty(conv, ctable.PrimaryKeys, "ALTER TABLE")
 					ctable.PrimaryKeys = []schema.Key{{Column: colname}}
 					conv.SrcSchema[tableName] = ctable
 				}
+				// TODO: Handle foreign key
```

Comment:
Unresolved TODO?

Reply:
This is also mentioned in PR note. It would just require 2 lines of code to append foreign keys but it's not updated because we wanted to check it using a sample mysqldump file.

Reply:
this is resolved, removed TODO and added handler for foreign keys